includes/connect/class-wc-stripe-connect.php

```diff
@@ -163,7 +163,7 @@ private function save_stripe_keys( $result, $type = 'connect', $mode = 'live' )
 			$options[ $prefix . 'publishable_key' ]     = $publishable_key;
 			$options[ $prefix . 'secret_key' ]          = $secret_key;
 			$options[ $prefix . 'connection_type' ]     = $type;
-
+			$options['pmc_enabled']                     = 'connect' === $type ? '' : 'no'; // When not connected via oauth, the PMC is disabled. Otherwise, set to empty string which will be set to 'yes' after the migration in 'maybe_migrate_payment_methods_from_db_to_pmc'.
```

Comment:
This feels like it might be risky: what happens if we connect in test mode, but toggle the main `pmc_enabled` flag? Should we just tackle using a prefixed version of the flag?

Reply:
All plugin settings (other than keys and webhooks) are shared between _live_ and _test_ mode; I think that having two Payment Method lists would be confusing 🤷🏼